packages/s3-credentials-endpoint/package.json

```diff
@@ -31,7 +31,8 @@
     "got": "^11.2.0",
     "hsts": "^2.2.0",
     "morgan": "^1.9.1",
-    "url-join": "^4.0.0"
+    "url-join": "^4.0.0",
+    "tough-cookie": "^4.0.0"
```

Comment:
This should be a dev-dependency in the project root package.json (we're hoisting dev dependencies for ~lerna reasons~) and not in `dependencies`

Reply:
Oh, okay, thanks! I'll keep that in mind for the future as well